ImageProcessingRequest_api

```diff
@@ -0,0 +1,86 @@
+v1.ImageProcessingRequest
+
+This is an object that describes request to the ChRIS system to be able to process an image into a desired format
+
+Fields:
+    - imageprocessingrequestid: an autogenerated id that corresponds to the submitted job
+    - patientid: The id of the patient
+    - jobtype: The type of image processing job you want to run
+    - jobstatus: The status of the job (e.g. 'Pending', 'Running', 'Failed', 'Success')
+
+
+
+
+create an ImageProcessingRequest
+
+POST /chrisapi/v1/imageprocessingrequests
+
+Parameters
```

Comment:
the pseudo-pfioh program I wrote for your demo uses it.  Django probably already has mechanisms for receiving it.  It's the encoding used by browsers when you submit any form (in this case, it would be most akin to submitting a file-upload form).

The mozilla developer center page [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) has a decent example of what the actual contents look like, but in the spec you can probably just mark those "fields" as part of a multiple/form-data body and leave it at that.